test/output/Issue-#29.json

```diff
@@ -17,6 +17,9 @@
           "author": {
             "$ref": "#/definitions/Author"
           },
+          "editors": {
+             "$ref": "#/definitions/Editors"
+          },
```

Comment:
`editors` is array of `Editors` and `Editors` is an object.
I think this `$ref` is wrong.

Reply:
So you're expecting to see something like:

```
"editors": {
    "type": "array",
    "items": {
        "$ref": "#/definitions/Editors"
    }
},
```
Okay, I'll add that :)